src/selectors/per-thread/markers.js

```diff
@@ -452,28 +451,12 @@ export function getMarkerSelectorsPerThread(
 
   /**
    * This selector is used by the generic marker context menu to decide what to copy.
+   * Currently we want to copy the same thing that is displayed as a description
+   * in the marker table.
    */
   const getMarkerLabelToCopyGetter: Selector<
```

Comment:
Note: I decided to keep this variable even if it's just a reference to another one, because I found it's clearer from the name about what it is. We may want to change it later to something else.